test/ingest_search/ingest_search_test.go

```diff
@@ -0,0 +1,235 @@
+package ingestsearchtest
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	splcommon "github.com/splunk/splunk-operator/pkg/splunk/common"
+	"github.com/splunk/splunk-operator/test/testenv"
+)
+
+var _ = Describe("Ingest and Search Test", func() {
+
+	var deployment *testenv.Deployment
+	var firstLine string
+
+	BeforeEach(func() {
+		var err error
+		deployment, err = testenvInstance.NewDeployment(testenv.RandomDNSName(3))
+		Expect(err).To(Succeed(), "Unable to create deployment")
+	})
+
+	AfterEach(func() {
+		// When a test spec failed, skip the teardown so we can troubleshoot.
+		if CurrentGinkgoTestDescription().Failed {
+			testenvInstance.SkipTeardown = true
+		}
+		if deployment != nil {
+			deployment.Teardown()
+		}
+	})
+
+	Context("Standalone deployment (S1)", func() {
+		It("can search internal logs for standalone instance", func() {
+
+			standalone, err := deployment.DeployStandalone(deployment.GetName())
+			Expect(err).To(Succeed(), "Unable to deploy standalone instance ")
+
+			// Wait for standalone to be in READY Status
+			testenv.StandaloneReady(deployment, deployment.GetName(), standalone, testenvInstance)
+
+			Eventually(func() splcommon.Phase {
+				podName := fmt.Sprintf("splunk-%s-standalone-0", deployment.GetName())
+
+				searchString := "index=_internal | stats count by host"
+				searchResultsResp, err := testenv.PerformSearchSync(podName, searchString, deployment)
+				if err != nil {
+					testenvInstance.Log.Error(err, "Failed to execute search on pod", "pod", podName, "searchString", searchString)
+					return splcommon.PhaseError
+				}
+
```

Comment:
Nit: Add a log here similar to L82?

Reply:
Added log mentioning successful search request.

Reply:
What I meant is a log for the sync as well just in case there is a failure or a delay here. But that works fine too :)